README.md

```diff
@@ -255,3 +255,14 @@ sub-dependencies. A sure way to solve it is to run:
 
     rm -fr node_modules
     yarn install
+
+### `Error: listen EADDRINUSE: address already in use :::5000`
+
+The default server port is conflicting with another running process
+(e.g., ControlCenter.app in macOS >= 12). To resolve this, you can pick any
+unused port (e.g., 6000) and run the following:
+
+```
+echo SERVER_PORT=6000 >> .env
+echo BUILD_LIVE_SAMPLES_BASE_URL=http://localhost:6000 >> .env
```

Comment:
Instead of this, I think you should edit `const LIVE_SAMPLES_BASE_URL =` in `kumascript/src/constants.js`. Something like this:
```diff
+const serverPort = process.env.SERVER_PORT || 5000;
+const serverURL = `http://localhost:${serverPort}`;
const LIVE_SAMPLES_BASE_URL =
  process.env.BUILD_LIVE_SAMPLES_BASE_URL !== undefined
    ? process.env.BUILD_LIVE_SAMPLES_BASE_URL
-   : "http://localhost:5000";
+   : serverURL; 
```

what do you think about that?

Reply:
Yes, this is a cleaner solution (we only need to update envvar). 

I just checked that this is the only place where we make use of `BUILD_LIVE_SAMPLES_BASE_URL`.  So I went ahead as you suggested.